File: MarketPortal/Market/Models/JWTRefreshResponse.cs
using Market.Data.Models;

namespace Market.Models
{
    public class JWTRefreshResponse
    {
        public JWTRefreshResponse(Token refreshToken, string accessToken)
        {
            RefreshToken = refreshToken;
            AccessToken = accessToken;
        }

        public Token RefreshToken { get; set; }
        public string AccessToken { get; set; }
    }
}

File: MarketFlutter/lib/models/dto/jwt_refresh_response.dart

import '../token.dart';


class JwtRefreshResponse{
  Token refreshToken;
  String accessToken;

  JwtRefreshResponse({required this.refreshToken, required this.accessToken});

  factory JwtRefreshResponse.fromJson(Map<String, dynamic> json) {
    JwtRefreshResponse res = JwtRefreshResponse(
      refreshToken: Token.fromJson(json['refreshToken']),
      accessToken: json['accessToken'] as String,
    );
    return res;
  }

  // Method to convert User instance to a JSON map
  Map<String, dynamic> toJson() {
    return {
      'refreshToken': refreshToken.toJson(),
      'accessToken': accessToken,
    };
  }
}